apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
```

Comment:
Can't quite remember but I don't think single field names need be declared in an array, they get cast to arrays when processed.